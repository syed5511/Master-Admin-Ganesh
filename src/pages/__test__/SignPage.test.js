import SignInPage from "../login";
import getSnapShot from "../../utils/getSnapShot";

test("Signin page snapshot", () => {
  getSnapShot(SignInPage)();
});
