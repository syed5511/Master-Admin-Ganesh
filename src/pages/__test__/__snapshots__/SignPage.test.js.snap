// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Signin page snapshot 1`] = `
<div
  className="sc-AxirZ cOqVN"
>
  <form
    className="sc-AxiKw gfDqWm form-signin"
  >
    <div
      className="text-center mb-4"
    >
      <h1
        className="h3 mb-3"
      >
        Sign In
      </h1>
      <p>
        Master Admin Kitchen Portal
      </p>
    </div>
    <div
      className="form-label-group"
    >
      <input
        autofocus={true}
        className="form-control"
        id="inputEmail"
        placeholder="Email address"
        required={true}
        type="email"
      />
      <label
        for="inputEmail"
      >
        Email address
      </label>
    </div>
    <div
      className="form-label-group"
    >
      <input
        className="form-control"
        id="inputPassword"
        placeholder="Password"
        required={true}
        type="password"
      />
      <label
        for="inputPassword"
      >
        Password
      </label>
    </div>
    <div
      className="checkbox mb-3"
    >
      <label>
        <input
          type="checkbox"
          value="remember-me"
        />
         Remember me
      </label>
    </div>
    <button
      className="sc-AxjAm jleUEH submit login-submit btn btn-primary"
      disabled={false}
      onClick={[Function]}
      type="submit"
    >
      Sign in
    </button>
  </form>
</div>
`;
